server/db/schemas/addresses.sql

```diff
@@ -1,11 +1,18 @@
 CREATE TABLE addresses (
-    address_id serial PRIMARY KEY,
-    user_id INT NOT NULL,
+    address_id SERIAL,
     street_address VARCHAR(255) NOT NULL,
-    street_address_line_two VARCHAR(255),
+    street_address_line_two VARCHAR(255) NOT NULL,
     city VARCHAR(255) NOT NULL,
     province VARCHAR(255) NOT NULL,
     postal_code VARCHAR(255) NOT NULL,
     country VARCHAR(255) NOT NULL,
-    FOREIGN KEY (user_id) REFERENCES users (user_id)
+    PRIMARY KEY (address_id),
+    UNIQUE (
+        street_address,
+        street_address_line_two,
+        city,
+        province,
+        postal_code,
+        country
+   )
```

Comment:
This was added so we can uniquely identify matching addresses without knowing the primary key.

Reply:
good idea